tests/Rector/Property/TypedPropertyFromToOneRelationTypeRector/Fixture/do_no_preprend_slash.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Rector\Property\TypedPropertyFromToOneRelationTypeRector\Fixture;
+
+
+use Doctrine\ORM\Mapping as ORM;
+use Acme\Entity\User;
+
+/**
+ * @ORM\Entity
+ */
+class DoNotPrependSlash
+{
+
+    /**
+     * @ORM\ManyToOne(targetEntity="\Acme\Entity\User", inversedBy="images")
```

Comment:
Acme\Entity\User seems different with original test, and class seems not exists, that's why it green here, it should not green if it not green in rector-src

Reply:
I stand corrected, the original tests seems was missing use statement, that's why it was error

https://github.com/rectorphp/rector-src/pull/4047/commits/3146b3277695e7db2ff212148a6f4c98e714925d#diff-9c450be000622c4ee6aa905e9e582ead8e3a054076f2c8b947a969ba67aed719

it seems ok and not need a fix for it, let's see if this issue happen again in the future.